scripts/hudson/narayana.sh

```diff
@@ -519,12 +530,16 @@ function rts_tests {
 }
 
 function lra_tests {
-  echo "#0. LRA Test"
+  if [ $LRA_WILDFLY_TESTS -eq 1 ]; then
+    echo "#0. LRA Tests with Wildfly"
+  else
+    echo "#0. LRA Tests with Thorntail"
+  fi
   ./build.sh install -pl ArjunaCore/arjunacore,rts -am -DskipTests -Pcommunity "$@"
   # we can't use 'mvn -f' option beacuse of Thorntail plugin issue THORN-2049
   cd ./rts/lra/
-  PRESERVE_WORKING_DIR=true ../../build.sh -fae -B -Pcommunity -P$ARQ_PROF $CODE_COVERAGE_ARGS $ENABLE_LRA_TRACE_LOGS -Dlra.test.timeout.factor="${LRA_TEST_TIMEOUT_FACTOR:-1.5}" "$@"
-  [ $? = 0 ] || fatal "LRA Test failed"
+  PRESERVE_WORKING_DIR=true ../../build.sh -fae -B -Pcommunity -P$ARQ_LRA_PROF $CODE_COVERAGE_ARGS $ENABLE_LRA_TRACE_LOGS -Dlra.test.timeout.factor="${LRA_TEST_TIMEOUT_FACTOR:-1.5}" "$@"
+  [ $? = 0 ] || fatal "LRA Tests"
```

Comment:
it would be good to inform in `fatal` what profile was executed and failed.

Reply:
The addition of these extra profiles is already causing confusion, even for the author of the PR, lol

Reply:
Answered above